Utils/Dataflow/071_esConsistency/consistency.py

```diff
@@ -151,6 +151,10 @@ def process(stage, message):
         log('Insufficient ES info in data:' + str(data), 'WARN')
         return False
 
+    _parent = None
+    if '_parent' in data:
+        _parent = data.pop('_parent')
```

Comment:
This will do as well (`pop()` with default value):
```
_parent = data.pop('_parent', None)
```

Reply:
Done, thanks.